evidencecore/intoto/v1_statement.go

```diff
@@ -0,0 +1,68 @@
+package intoto
+
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+
+	"github.com/jfrog/jfrog-client-go/artifactory"
+)
+
+const DssePayloadType = "application/vnd.in-toto+json"
+const IntotoType = "https://in-toto.io/Statement/v0.1"
+
+type Statement struct {
+	Type          string          `json:"_type"`
+	Subject       []Descriptor    `json:"subject"`
+	PredicateType string          `json:"predicateType"`
+	Predicate     json.RawMessage `json:"predicate"`
```

Comment:
Looks like this field is optional - https://github.com/in-toto/attestation/tree/v0.1.0/spec#statement
```suggestion
	Predicate     json.RawMessage `json:"predicate,omitempty"`
```